pyocd/trace/swv.py

```diff
@@ -49,10 +49,7 @@ def receive(self, event):
         """
         if not isinstance(event, TraceITMEvent):
             return
-        
-        if not event.port == 0:
-            return
-        
+
```

Comment:
Fwiw, this restriction to port 0 was present because only port 0 is defined as the SWV stdio-style logging port. Instead of fixing it to port 0, we could make the port configurable.

The problem is that SWO reading and SWV processing are combined into a single class. Ultimately the intention is to migrate to a better design with a generic SWO reader thread that accepts a data flow graph for processing events, and SWV processing is done in a graph sink class.

Reply:
The use of a particular port for a particular purpose is entirely based on convention, not definition.  Making it configurable would be fine, though I would leave that to subsequent work.